### PR TITLE
Fixed: radio buttons

### PR DESCRIPTION
**Pull request by richerry**
Modified: Your Details page now works going backwards
Modified: Improved formatting in certain areas
Updated: Django to 1.11.3

### CODE REVIEW

**Comment by richerry** [comment]
Looking for review, available to rebase, not ready to merge yet (need to review it myself).

File: apps/forms/templates/widgets/plea_radio_field.html [hunk 1]
```diff
@@ -5,90 +5,131 @@
...
-<div class="form-group-wide{% if form.guilty.errors %} with-error{% endif %}" id="section_guilty">
+<div class="form-group-wide{% if form.guilty.errors %} with-error{% endif %}"
+     id="section_guilty">
 
     <fieldset role="radiogroup" aria-labelledby="label_charge">
 
         <legend>
+
             <div class="header-line" id="label_charge">
-                <span class="label-text">{% spaceless %}
```
Comment (robyoung) [inline apps/forms/templates/widgets/plea_radio_field.html]: Is there a reason we've gotten rid of the `{% spaceless %}` here?

File: apps/forms/templates/widgets/plea_radio_field.html [hunk 2]
```diff
@@ -5,90 +5,131 @@
...
+                        <label for="id_{{ choice.data.name }}_{{ choice.data.value }}"
+                               class="block-label">
+                            <input id="id_{{ choice.data.name }}_{{ choice.data.value }}"
+                                   type="radio"
+                                   name="{{ choice.data.name }}"
+                                   value="{{ choice.data.value }}"
+                                   {% if choice.data.selected %}checked="checked"{% endif %}>
+                            {{ choice.choice_label }}
+                        </label>
+                    {% endfor %}
```
Comment (robyoung) [inline apps/forms/templates/widgets/plea_radio_field.html]: Could this be implemented with the `DSRadioSelect` partial?

File: apps/plea/admin.py [hunk 3]
```diff
@@ -371,8 +371,14 @@ def has_add_permission(self, request):
         return False
 
 
+class OffenceAdmin(admin.ModelAdmin):
```
Comment (robyoung) [inline apps/plea/admin.py]: Do we need to add this? Given that this Django admin is exposed to admin staff it feels like we should be keeping things out of it unless absolutely necesarry for those admin staff.

File: apps/forms/fields.py [hunk 4]
```diff
@@ -7,29 +7,27 @@
 from django.utils.translation import ugettext_lazy as _
 
 
+class DSPleaRadioSelect(RadioSelect):
```
Comment (robyoung) [inline apps/forms/fields.py]: Why do we need another type here?

File: apps/forms/templates/widgets/plea_radio_field.html (hunk 1 above)
Comment (richerry) [inline apps/forms/templates/widgets/plea_radio_field.html]: Didn't seem to be doing anything useful. If we can avoid needing it that would be good.

File: apps/forms/templates/widgets/plea_radio_field.html (hunk 2 above)
Comment (richerry) [inline apps/forms/templates/widgets/plea_radio_field.html]: Done. Well, I'm not sure the template's not used but the form now just uses dsradioselect.

**Pull request merged**